converter/src/test/java/gov/cms/qpp/conversion/decode/QualityMeasureScopedTest.java

```diff
@@ -9,57 +9,57 @@
 import gov.cms.qpp.conversion.model.validation.SubPopulations;
 import gov.cms.qpp.conversion.segmentation.QrdaScope;
 import gov.cms.qpp.conversion.xml.XmlException;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
 import java.io.IOException;
 import java.nio.file.Paths;
 import java.util.Arrays;
 import java.util.stream.Stream;
 
-import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
-public class QualityMeasureScopedTest {
+class QualityMeasureScopedTest {
 	private static String location = "src/test/resources/fixtures/qppct298/cms137v5.xml";
 
 
 	@Test
-	public void internalDecodeValidMeasure137V5Ipop() throws IOException, XmlException {
+	void internalDecodeValidMeasure137V5Ipop() throws IOException, XmlException {
 		Node result = scopedConversion(QrdaScope.MEASURE_REFERENCE_RESULTS_CMS_V2, location);
 		long ipops = pluckDescendants(result, TemplateId.MEASURE_REFERENCE_RESULTS_CMS_V2, TemplateId.MEASURE_DATA_CMS_V2)
 				.filter(node -> node.getValue(MeasureDataDecoder.MEASURE_TYPE).equals(SubPopulations.IPOP))
 				.count();
 
-		assertEquals("Valid CMS137v5 measure should have 2 IPOP values", ipops, 2);
+		assertEquals(ipops, 2);
```

Comment:
We should convert this, and the other lines below, to Google Truth.

Reply:
Good catch